modules/pruning-images.adoc

```diff
@@ -5,111 +5,152 @@
 [id="pruning-images_{context}"]
 = Pruning images
 
-In order to prune images that are no longer required by the system due to age,
-status, or exceed limits, administrators can run the following command:
+Administrators can manually prune images that are no longer required by the system due to age, status, or exceed limits. There are two methods to manually prune images:
 
+* Running image pruning as a `Job` or `CronJob` on the cluster.
+* Running the `oc adm prune images [<options>]` command.
+
+.Prerequisites
+
+* To manually prune images, you must first log in to the CLI as a user with an access token. The user must also have the cluster role of `system:image-pruner` or greater (for example, `cluster-admin`).
```

Comment:
s/cluster role of `system:image-pruner` or greater/`system:image-pruner` cluster role or greater ??

Reply:
Do we need _To manually prune images,_ here? 
Seems a bit confusing to me. Sounds like you need this role for one of the two methods, which isn't true(?).